source/android/adaptivecards/src/main/res/values/attrs.xml

```diff
@@ -7,4 +7,9 @@
     <attr name="adaptiveShowCardAction" format="reference" />
     <style name="adaptiveInlineAction" format="reference" />
     <style name="adaptiveInlineActionImage" format="reference" />
+
+    <attr name="usingCustomInputs" format="boolean" />
+    <declare-styleable name="custom_input_state">
+        <attr name="state_input_invalid" format="boolean" />
```

Comment:
How about "state_error" to keep it generic?

Reply:
I like the idea but I'm having a tough time thinking of other scenarios where we may use the error state